src/main/scala/com/timeout/healthchecks/routes/HealthRoutes.scala

```diff
@@ -0,0 +1,33 @@
+package com.timeout.healthchecks.routes
+
+import akka.http.scaladsl.model.StatusCodes._
+import akka.http.scaladsl.server.Directives._
+import de.heikoseeberger.akkahttpcirce.CirceSupport
+import io.circe.generic.JsonCodec
+import com.timeout.healthchecks._
+import cats.syntax.all._
+import cats.std.all._
+
+import scala.collection.convert.DecorateAsScala
+import scala.concurrent.{ExecutionContext, Future}
+
+object HealthRoutes extends CirceSupport with DecorateAsScala {
+
+  @JsonCodec case class HealthResult(status: String, checks: Seq[String], failures: Seq[String])
+
+  def status(s: Boolean) = if (s) "OK" else "KO"
```

Comment:
purely for backward compatibility reasons (graffiti convention and zabbix settings), could we have it either "healthy" or "unhealthy" ?